src/functions/keycloak/update-user-vacation/index.ts

```diff
@@ -0,0 +1,16 @@
+import { handlerPath } from "@libs/handler-resolver";
+
+export default {
+  handler: `${handlerPath(__dirname)}/handler.main`,
+  events: [
+    {
+      httpApi: {
+        method: "put",
+        path: "/users/{userId}/vacation",
```

Comment:
I'm not certain about this endpoint name. Might vacationDays or vacationDaysRecord be better/ clearer?

Reply:
Changing the name will also require spec updates